### ISSUE TITLE
Custom fields - pages / Modsecurity

### ISSUE THREAD

**Issue by Alex-e107nl** [e107inc/e107#4889]
I use the custom fields in pages (addittional) on 1 e107 website but I get a server error when I try to change existing pages, modsecurity intervenes with the following message: POST /e107_admin/cpage.php?action=edit&id=217 HTTP/ 2.0 Rule id 211050 maybe someone else had this too?

**Comment by Alex-e107nl** [comment]
yes its always page content, when using an iframe, loading script from somewhere else, wrong utf character combination, it gives me much problems in Tiny mce, isn't there a solution, loggedin as admin, Modsecurity off :-)

**Comment by CaMer0n** [comment]
Well, you could just disable rule 211050 if that is the only one causing you an issue. ?

**Comment by Alex-e107nl** [comment]
Yeah i had to disable rules 212320 - 212740 - 212280 - 211050 and 210230 to get it to work normally... some e107 websites are years and years old and updated to the latest git now and in that process i had to disable that rules...
